homeassistant/components/iammeter/const.py

```diff
@@ -0,0 +1,219 @@
+    ),
+    IammeterSensorEntityDescription(
+        key="Current_B",
+        name="Current_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Power_B",
+        name="Power_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ImportEnergy_B",
+        name="ImportEnergy_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ExportGrid_B",
+        name="ExportGrid_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Frequency_B",
+        name="Frequency_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfFrequency.HERTZ,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="PF_B",
+        name="PF_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement="",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Voltage_C",
+        name="Voltage_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Current_C",
+        name="Current_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Power_C",
+        name="Power_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ImportEnergy_C",
+        name="ImportEnergy_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ExportGrid_C",
+        name="ExportGrid_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Frequency_C",
+        name="Frequency_C",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfFrequency.HERTZ,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
```

Comment:
- Device class missing
- We should disable this sensor by default
- missing entity category?